interviewQuestions/MaximumsumProblem.java

```diff
@@ -0,0 +1,34 @@
+/*Recursively break a number in 3 parts to get maximum sum
```

Comment:
@swatiprajapati08 make the comment style like this and also fix some grammatical mistakes
```java
/**
 * Recursively break a number in 3 parts to get maximum sum.
 * Given a number n, we can divide it into only three parts n/2, n/3 and n/4 (we will consider only integer part).
 * The task is to find the maximum sum we can make by dividing the number into three parts recursively and summing up them together.
 * https://practice.geeksforgeeks.org/problems/maximum-sum-problem/0 
 **/
```

Reply:
Also, refactored the code according to Google Java Style Guide